src/thing/mock.test.ts

```diff
@@ -22,12 +22,18 @@
 import { describe, it, expect } from "@jest/globals";
 
 import { mockThingFrom } from "./mock";
-import { asIri } from "./thing";
+import { asIri, isThing } from "./thing";
 
 describe("mockThingFrom", () => {
-  it("is is identified by the given IRI", async () => {
+  it("is identified by the given IRI", async () => {
     const mockedThing = mockThingFrom("https://some.pod/resource#thing");
 
     expect(asIri(mockedThing)).toBe("https://some.pod/resource#thing");
   });
+
+  it("is is recognised as a Thing", async () => {
```

Comment:
```suggestion
  it("is recognised as a Thing", async () => {
```
In memory of the one you fixed above :p

Reply:
Stupid copy-pasting me.

Reply:
Shall he who has never sinned cast the first stone